Library/Homebrew/dev-cmd/bump.rb

```diff
@@ -239,10 +256,17 @@ def retrieve_and_display_info(formula_or_cask, name, repositories, args:, ambigu
 
     ohai title
     puts <<~EOS
-      Current #{formula_or_cask.is_a?(Formula) ? "formula version:" : "cask version:   "}  #{current_version}
+      Current #{version_name}:  #{current_version}
       Latest livecheck version: #{livecheck_latest}
       Latest Repology version:  #{repology_latest}
+      Open pull requests:       #{pull_requests || "none"}
     EOS
-    puts "Open pull requests:       #{pull_requests}" unless pull_requests.nil?
+
+    return unless args.open_pr?
+    return unless new_version
+    return if pull_requests
+
+    system HOMEBREW_BREW_FILE, "bump-#{type}-pr", "--no-browse",
+           "--message=Created by `brew bump`", "--version=#{new_version}"
```

Comment:
```suggestion
           "--message=Created by `brew bump`", "--version=#{new_version}", name
```

Did some final testing and noticed this won't work if we don't provide the formula/cask name.